cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -101,6 +107,17 @@ func (m *GceManager) RegisterMig(mig *Mig) {
 	m.migs = append(m.migs, &migInformation{
 		config: mig,
 	})
+
+	template, err := m.getMigTemplate(mig)
+	if err != nil {
+		glog.Errorf("Failed to build template for %s", mig.Name)
+	}
+
+	node, err := m.buildNodeFromTemplate(mig, template)
```

Comment:
Are you building this here to save it and return from Mig.TemplateNodeInfo()? If so why not include that part in this PR?